packages/shared/src/lib/cookie.ts

```diff
@@ -97,47 +61,3 @@ export const expireCookie = (
 
   setCookie(name, 'expired', { path, domain, maxAge: 0 });
 };
-
-export const updateFeatureFlags = (
```

Comment:
We will have to come up another way of overriding feature flags for preview deployment but have decided to take it into a different PR as this is a large one already.